CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -97,7 +97,7 @@ public async Task<ActionResult<IList<StudentLessonDto>>> GetStudentLessons(long
         /// Get only active students
         /// </summary>
         /// <response code="200">Successful return of students list</response>
-        [Authorize(Roles = "Admin, Mentor, Secretary")]
+        [Authorize(Roles = "Admin, Mentor, Secretary, Student")]
```

Comment:
It seems to me not very correct to give a student access to information about all active students. Topic for discussion.